app/views/provider_interface/application_choices/show.html.erb

```diff
@@ -28,6 +28,10 @@
   </div>
 </div>
 
+<% unless HostingEnvironment.production? %>
+  <%= button_to 'Sign in as this candidate (sandbox mode)', provider_interface_impersonate_candidate_path(@application_choice.application_form.candidate), class: 'govuk-button' %>
```

Comment:
I think we can lose "Sandbox mode" here as it's not the case any more? (It's still available when `SANDBOX` is off)

Reply:
I would like to say something to clarify that they shouldn't expect to see this in production. How about 'test mode'?

Reply:
"Test environments only"?